<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameWindow</class>
 <widget class="QMainWindow" name="gameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>450</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>quit</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>621</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="wheelButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>400</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>spin</string>
    </property>
   </widget>
   <widget class="QLabel" name="wheelOutput">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>340</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Action</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="selectStartRadioButton">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>30</y>
       <width>211</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>select the pawn to move</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="sinkRadioButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>161</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>sink tile</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="selectDestinationRadioButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>221</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>select the tile to move onto</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
